integreat_cms/google_translate_api/apps.py

```diff
@@ -0,0 +1,134 @@
+                self.supported_source_languages += [language["language"]]
+                self.supported_target_languages += [language["language"]]
+
+            assert self.supported_source_languages
+            logger.debug(
+                "Supported source languages by Google Translate: %r",
+                self.supported_source_languages,
+            )
+            assert self.supported_target_languages
+            logger.debug(
+                "Supported target languages by Google Translate: %r",
+                self.supported_target_languages,
+            )
+
+            logger.info("Google Translate API is enabled.")
+        except AssertionError as e:
+            logger.error(e)
+            logger.error("Google translate is not available")
+
+    def ready_v3(self, credentials: Credentials) -> None:
+        """
+        Preparing Google Translate with advanced version
+        """
+        logger.info(translate_v3)
+        try:
+            google_translator_v3 = translate_v3.TranslationServiceClient(
+                credentials=credentials
+            )
+            parent = settings.GOOGLE_PARENT_PARAM
+
+            supported_languages = google_translator_v3.get_supported_languages(
+                parent=parent
+            ).languages
+
+            self.supported_source_languages = []
+            self.supported_target_languages = []
+
+            for language in supported_languages:
+                if language.support_source:
+                    self.supported_source_languages += [language.language_code]
+                if language.support_target:
+                    self.supported_target_languages += [language.language_code]
+
+            assert self.supported_source_languages
+            logger.debug(
+                "Supported source languages by Google Translate: %r",
+                self.supported_source_languages,
+            )
+            assert self.supported_target_languages
+            logger.debug(
+                "Supported target languages by Google Translate: %r",
+                self.supported_target_languages,
+            )
+
+            logger.info("Google Translate API is enabled.")
+        except AssertionError as e:
+            logger.error(e)
+            logger.error("Google translate is not available")
+
+    def ready(self) -> None:
+        """
+        Checking if API is available
+        """
+        # Only check availability if running a server
+        if "runserver" in sys.argv or "APACHE_PID_FILE" in os.environ:
+            if settings.GOOGLE_TRANSLATE_ENABLED:
+                try:
+                    credentials = service_account.Credentials.from_service_account_file(
+                        settings.GOOGLE_APPLICATION_CREDENTIALS
+                    )
+                    if settings.GOOGLE_TRANSLATE_VERSION == "Advanced":
+                        self.ready_v3(credentials)
+                    else:
+                        self.ready_v2(credentials)
```

Comment:
```suggestion
                        self.ready_v2(credentials)
                        
                        assert self.supported_source_languages
                        logger.debug(
                            "Supported source languages by Google Translate: %r",
                            self.supported_source_languages,
                        )

                        assert self.supported_target_languages
                        logger.debug(
                            "Supported target languages by Google Translate: %r",
                            self.supported_target_languages,
                        )

                        logger.info("Google Translate API is enabled.")
```

I'd suggest moving language logging here, because it's the same code for both versions